docs/conf.py

```diff
@@ -12,9 +12,9 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
 import os
 import shlex
+import sys
```

Comment:
I didn't want to touch this, `isort` was complaining about it, and wouldn't shut up.